modules/client/src/test/java/org/apache/ignite/client/ClientTupleTest.java

```diff
@@ -94,16 +96,16 @@ public void testEmptySchemaThrows() {
     @Test
     public void testSetThrowsWhenColumnIsNotPresent() {
         var ex = assertThrows(IgniteException.class, () -> getBuilder().set("x", "y"));
```

Comment:
BTW, should we assert a more specific type here? We do have a right subclass here